src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/SignatureResourcePool.cs

```diff
@@ -0,0 +1,123 @@
+using System.IO;
+using System.Security.Cryptography;
+using CoreWCF.IdentityModel.Tokens;
+using System.Xml;
+using System.Text;
+
+namespace CoreWCF.IdentityModel
+{
+    // for sequential use by one thread
+    sealed class SignatureResourcePool
+    {
```

Comment:
I believe this class is used by the IdentityModel version of SignedXml and isn't needed.